test/teslamate/mqtt/pubsub/vehicle_subscriber_test.exs

```diff
@@ -145,7 +146,8 @@ defmodule TeslaMate.Mqtt.PubSub.VehicleSubscriberTest do
       ideal_battery_range_km: 230.52,
       rated_battery_range_km: 230.52,
       scheduled_charging_start_time: DateTime.utc_now() |> DateTime.add(60 * 60 * 10, :second),
-      time_to_full_charge: 2.5
+      time_to_full_charge: 2.5,
+      charge_rate: 123
```

Comment:
I wonder if 123 is a realistic charge rate

Reply:
As vefore